src/drawings/circle/Circle.jsx

```diff
@@ -3,26 +3,26 @@
 import { CanvasBuilder } from '@/components/Canvas'
 import { Color } from '@/constants'
 
+const HEIGHT = 100
+const WIDTH = 100
```

Comment:
Providing dimensions here aligns with my goal of having each drawing own all off its config